internal/xds/translator/testdata/out/xds-ir/port-as-listener-name.listeners.yaml

```diff
@@ -0,0 +1,371 @@
+- address:
+    socketAddress:
```

Comment:
This test output shows the name schemes for listener, route, filterChain, and statPrefix.